### ISSUE TITLE
Context reference checks are including documents in Reference_List

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS-Incubator/validate#450]
## 🐛 Describe the bug

When running against Mars2020 PIXL data, getting invalid `warning.integrity.missing_context_reference` warnings.

```
  PASS: file:/Users/jpadams/test/urn-nasa-pds-mars2020_pixl/bundle_pixl.xml
      WARNING  [warning.integrity.missing_context_reference]   This file should reference 'urn:nasa:pds:mars2020_pixl:document:pixl_bundle_sis' because its child product with LIDVID urn:nasa:pds:mars2020_pixl:data_processed:ps__0140_0679334390_000rbs__00518120532812811688___j02::1.0 references it.
      WARNING  [warning.integrity.missing_context_reference]   This file should reference 'urn:nasa:pds:mars2020_pixl:document:pixl_edr_sis' because its child product with LIDVID urn:nasa:pds:mars2020_pixl:data_processed:ps__0140_0679334390_000rbs__00518120532812811688___j02::1.0 references it.
      WARNING  [warning.integrity.missing_context_reference]   This file should reference 'urn:nasa:pds:mars2020_pixl:document:pixl_rdr_sis' because its child product with LIDVID urn:nasa:pds:mars2020_pixl:data_processed:ps__0140_0679334390_000rbs__00518120532812811688___j02::1.0 references it.
        2 integrity check(s) completed
```

## 📜 To Reproduce
1. From pds-dev3, run validate like so:
```
validate -R pds4.bundle -t /data/pds4/test-data/registry/urn-nasa-pds-mars2020_pixl/bundle_pixl.xml
```

## 🕵️ Expected behavior

The product it is throwing a warning about has the following:
```
  <Reference_List>
    <Internal_Reference>
      <lid_reference>urn:nasa:pds:mars2020_pixl:document:pixl_bundle_sis</lid_reference>
      <reference_type>data_to_document</reference_type>
    </Internal_Reference>
    <Internal_Reference>
      <lid_reference>urn:nasa:pds:mars2020_pixl:document:pixl_edr_sis</lid_reference>
      <reference_type>data_to_document</reference_type>
    </Internal_Reference>
    <Internal_Reference>
      <lid_reference>urn:nasa:pds:mars2020_pixl:document:pixl_rdr_sis</lid_reference>
      <reference_type>data_to_document</reference_type>
    </Internal_Reference>
```

That should not be included in the check for context references.

---

## 🦄 Related requirements
- 🦄 https://github.com/nasa-pds/validate/issues/69

**Comment by jordanpadams** [comment]
closed per #451

**Comment by jordanpadams** [comment]
Tagging as invalid since this is related to #69, which has been reverted
